r-python-eds-lessons/setup-r-envt.R

```diff
@@ -0,0 +1,26 @@
+##################
+# This code installs the needed R packages into the build environment
+# Author: Leah A. Wasser, Gina L. Li, Joseph V. Tuccillo
+# last modified: May 2019
+##################
+
+packages <- c("ggplot2", "dplyr", "RJSONIO",
```

Comment:
This is somewhat hard to read and at a glance see whether a particular package is installed. Alphabetizing the package order would be helpful.

Reply:
@mbjoseph I think the package order might matter here. I tried alphabetizing the vector of package names and multiple installs failed.

Reply:
@mbjoseph The order seems to matter here -- the build breaks on installing `ggplot2`, `ggmap`, `widyr` when I alphabetize the package names.

Reply:
Scratch that, upon further inspection, the problem seems to be due to a missing dependency (`nlme`)...so probably related more closely to the conversation just above.